### PR TITLE
Update release notes & fix various small typos & mistakes

### PR DESCRIPTION
**Pull request by havan**
* fix typos & small errors
* update camino-node release notes

### CODE REVIEW

File: docs/apps/nodes/add-validator.md [hunk 1]
```diff
@@ -496,7 +496,7 @@ curl -s -X POST --data '{
 - **endTime**: The time that the validation period ends. It is the same format as `startTime`. In this example we used _2 days_ in the future.
   For Camino mainnet, you need minimum **2 weeks** and maximum **365 days** for the `endTime`.
 
-- **stakeAmount**: This is the _**fixed**_ amount of tokens to stake. For Camino testnet `columbus` it is `2000000000000` nCAMs (2000 CAMs). For mainnet it is `100000000000000` nCAMs (100k CAMs).
+- **stakeAmount**: This is the _**fixed**_ amount of tokens to stake. For Camino mainnet `camino` and testnet `columbus` it is `100000000000000` nCAMs (100k CAMs). (For devnet `kopernikus` it is 2k CAMs)
```
Comment (mo-c4t) [inline docs/apps/nodes/add-validator.md:499]: should we give information about devnet in the public docs? I don't know but maybe ask Alex or Marcus.

File: docs/apps/release-notes/camino-node.md [hunk 2]
```diff
@@ -4,11 +4,97 @@ sidebar_position: 1
...
+[View on GitHub](https://github.com/chain4travel/camino-node/releases/tag/v0.4.1-rc2)
+
+- Split genesis `platformAllocation` to only stake `maxStakingAmount`
+- Add depositTx inputs for already bonded UTXOs
+
+**Full Changelog**: https://github.com/chain4travel/camino-node/compare/v0.4.1-rc1...v0.4.1-rc2
+
+---
+
+## v0.4.1-rc1
```
Comment (mo-c4t) [inline docs/apps/release-notes/camino-node.md:26]: This version doesn't work, shouldn't we remove it or maybe write a warning beside it ?

File: docs/apps/nodes/add-validator.md (hunk 1 above)
Comment (havan) [reply docs/apps/nodes/add-validator.md:499]: When you run local network for testing/developing etc, having the info is handy. Also I don't see what harm could it do. @Noctunus @peak3d any comments?

File: docs/apps/release-notes/camino-node.md (hunk 2 above)
Comment (havan) [reply docs/apps/release-notes/camino-node.md:26]: The page is "release-notes", so I think should stay but we can put a warning to not use it. I will also plan to make a page where it shows the current _recommended_ node version for mainnet & testnet.

File: docs/about/columbus-testnet.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ For on-chain development like deploying and testing solidity-based smart-contrac
 
 ## Metamask configuration
 
-For a [metamask](https://metamask.io/) configuration simply choose a network name, use the RPC-URL <https://columbus.camino.foundation/ext/bc/C/rpc>, Chain-ID `502`, Currency-Symbol `CAM` and the Block-Explorer URL <https://explorer.camino.foundation>.
+For a [metamask](https://metamask.io/) configuration simply choose a network name, use the RPC-URL <https://columbus.camino.foundation/ext/bc/C/rpc>, Chain-ID `501`, Currency-Symbol `CAM` and the Block-Explorer URL <https://explorer.camino.foundation>.
```
Comment (c4t-ag) [inline docs/about/columbus-testnet.md:20]: Worth to mention that we are already on chainlist.org which makes MM init easier.....

File: docs/apps/nodes/add-validator.md [hunk 4]
```diff
@@ -54,8 +54,8 @@ You will need:
   Note that this actually an address, just like a reqular wallet address.
 - **PrivateKey** of your node. <br/>
   You will need this to prove that you own the **NodeID**. Keep it safe!
-- **2k CAMs** in your P-Chain wallet. <br/>
-  Check from online [wallet](https://wallet.camino.network) that you have the at least 2000 tokens on P-Chain.
+- **100k CAMs** in your P-Chain wallet. <br/>
```
Comment (c4t-ag) [inline docs/apps/nodes/add-validator.md:57]: 100K unlocked CAMS

File: docs/apps/nodes/add-validator.md [hunk 5]
```diff
@@ -310,7 +310,7 @@ _**You should only create a keystore user on a node that you operate, as the nod
 
 ### Import Wallet & Node Private Keys
 
-In order to prove that we own the wallet and the node, we need to add our prive keys into the node's keystore.
+In order to prove that we own the wallet and the node, we need to add our private keys into the node's keystore.
```
Comment (c4t-ag) [inline docs/apps/nodes/add-validator.md:313]: In the sentence oboce we are using "you", and here "we", I suggest that the complete documentation is "you"

File: docs/apps/nodes/add-validator.md (hunk 1 above)
Comment (c4t-ag) [reply docs/apps/nodes/add-validator.md:499]: I think it's OK to leave it for the devs

**Review by c4t-ag — CHANGES_REQUESTED** [review]

File: docs/about/columbus-testnet.md (hunk 3 above)
Comment (havan) [reply docs/about/columbus-testnet.md:20]: Fixed

File: docs/apps/nodes/add-validator.md (hunk 4 above)
Comment (havan) [reply docs/apps/nodes/add-validator.md]: Fixed

File: docs/apps/nodes/add-validator.md (hunk 5 above)
Comment (havan) [reply docs/apps/nodes/add-validator.md]: Fixed

File: docs/apps/release-notes/camino-node.md (hunk 2 above)
Comment (havan) [reply docs/apps/release-notes/camino-node.md:26]: Added a warning

**Review by c4t-ag — APPROVED** [review]

**Pull request merged**